PatientRegistrationApp/src/App.vue

```diff
@@ -124,17 +124,17 @@
 								<v-toolbar-title>Enter your contact information</v-toolbar-title>
 							</v-toolbar>
 							<v-card flat>
-								<SinglePatientContactInfo/>
+								<SinglePatientContactInfo ref="singlepatientcontactinfo"/>
 							</v-card>
 							<v-card-actions>
 								<v-icon large color="secondary"
-									@click="page=2"
+									@click="goToPage(2)"
```

Comment:
Will you implement the constant/enum thing for 2 being explicit later or is that part of this card?

Reply:
This is being done as part of MVS-301

Reply:
This is being done as part of MVS-301